src/js/components/project-view/clone-instructions.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+
+import Icon = require('react-fontawesome');
+
+import { Project } from '../../modules/projects';
+import { selectText } from './helpers';
+
+const styles: Styles = require('./setup-instructions.scss');
+
+interface Styles {
+  instructions: string;
+  section: string;
+  label: string;
+  heading: string;
+  'heading-icon': string;
+  text: string;
+  code: string;
+  url: string;
+}
```

Comment:
I'd rather add this to the UI quite soon, but yeah I can stash it somewhere for now